python/python/glide/async_commands/server_modules/vss.py

```diff
@@ -0,0 +1,21 @@
+from typing import List, cast
+from glide.glide_client import TGlideClient
+from glide.constants import TOK, TEncodable
+from glide.protobuf.command_request_pb2 import RequestType
+from glide.async_commands.server_modules.vss_options import FtCreateOptions, FtSearchOptions
+
+async def create(
+    client: TGlideClient,
+    indexName: TEncodable,
+    options: FtCreateOptions
+) -> TOK:
+    args: List[TEncodable] = [indexName]
+    args.extend(options.getCreateOptions())
+    return cast(TOK, await client._execute_command(RequestType.FtCreate, args))
```

Comment:
take a look at the implemented json commands, we use `custom_command` :)

Reply:
the reason is that when you will implement the same command in node/ java , you wont be able to use `_execute_command` without changing it to be public

Reply:
In Java, this won't be a problem, since we have `submitNewCommand` that is already designed to accommodate this.  

In Node, this might be a problem...